Changed the layout to center anchors in the middle of the screen

radiantdinosaurs opened the pull request:
### Acknowledgments

- [ ] I confirm that I wrote the code based on the [company guidelines](https://github.com/0x4447/0x4447-Foundation/tree/master/programmer#coding-principles).
- [ ] I confirm that I followed the Git flow described in the [company guidelines](https://github.com/0x4447/0x4447-Foundation/tree/master/programmer#how-to-work-with-github).

### Description

Changed the rows of the anchors to be the full height of the screen and removed the featurette divider, which was originally setting them apart with a margin. By doing this, we get each anchor section to be full screen on most screen sizes except phones, where the content will stretch the `div`.

Also, I minorly adjusted the size of the `vh` for the containers used at the top of full height containers.

Since we no longer needed the dividers, I removed them and attached the `id` to the `row` containing the anchor. Also, on mobile, I made sure to set a margin on the rows to keep them apart from each other a little bit.

Issue(s): #58

radiantdinosaurs commented:
Just to note -- I had changed the settings of my editor recently, but it still meets the standards (e.g., converting tabs to spaces, etc.). But, let me know if anything looks off.

davidgatti commented:
@radiantdinosaurs yep, you are saving with spaces and not tabs :) double check your IDE, and also make sure before you save to display white characters (each IDE have an option like that) do check what is actually happening, this way we don't loose track fo what actually changed, since converting between the two way of indentation is a regular change, and git will mark the whole file as a change, and you loos the actually difference that was done. 

Right now I did went over each file and made it 100% tabs, so we start fresh :)

radiantdinosaurs commented:
I thought we were supposed to be translating tabs to spaces, as said in the guidelines. I changed it to go from spaces to tabs now, though. I also removed my newer setting to auto-format on save, so this shouldn't be a problem anymore.
